rust/private/rustc.bzl

```diff
@@ -191,12 +191,13 @@ def get_cc_user_link_flags(ctx):
     """
     return ctx.fragments.cpp.linkopts
 
-def get_linker_and_args(ctx, attr, cc_toolchain, feature_configuration, rpaths):
+def get_linker_and_args(ctx, attr, toolchain, cc_toolchain, feature_configuration, rpaths):
```

Comment:
I could, but I tried to match other functions that use both the Rust toolchain and the CC toolchain. I don't feel strongly here.

Reply:
I think this would fall under https://github.com/bazelbuild/rules_rust/issues/889 where some concerns were raised about passing larger structs (like `ctx.attr` and `ctx.fragments`) to functions. Unless something out of our control or difficult to refactor requires that the entire toolchain be passed, I think the desire is to pass smaller components. Maybe @hlopko can speak to that 😄

Reply:
Once again, don't feel strongly, but I do think passing the toolchain here is better so that the arguments are well-documented and type-checked.

Reply:
Yeah, it's fine to leave the PR as is. I think the toolchain won't cause issues in existing aspects. Happy to continue the conversation on the other ticket as well.